tests/unit/steps/task/test_quality_scorer.py

```diff
@@ -42,15 +45,15 @@ def test_format_input(self) -> None:
         [
             (
                 "[1] Score: 1\n[2] Score: 2\n[3] Score: 3\n",
-                {"quality_score": [1.0, 2.0, 3.0]},
+                {"quality_scores": [1.0, 2.0, 3.0]},
```

Comment:
If you happen to apply the change above to rename `quality_scores` to `scores`, then it should be updated within this file too 👍🏻